main.js

```diff
@@ -0,0 +1,92 @@
+var cities = [];
+var citiesForecast = [];
+
+$('.search').on('click', function () {
+  var search = $('#search-query').val();
+  fetch(search);
+});
+
+var addCities = function (data) {
+  cities = [];
+
+  var city = {
+    temp: Math.ceil(data.main.temp),
+    city: data.name,
+    conditions: data.weather[0].main,
+    thumbnailURL: data.weather[0].icon
+  };
+  cities.push(city);
+  renderCities();
+};
+
+var fetch = function (query) {
+  $.ajax({
+    method: "GET",
+    url: "https://api.openweathermap.org/data/2.5/weather?q=" + query + "&appid=fde555c101a70d39eb7ec797d3514bce&units=imperial",
+    dataType: 'json',
+    success: function (data) {
+      addCities(data);
+    },
+    error: function (jqXHR, textStatus, errorThrown) {
+      console.log(textStatus);
+    }
+  });
+};
+
+var renderCities = function () {
+  $('.cities').empty();
+  for (var i = 0; i < cities.length; i++) {
+    var source = $('#weather-template').html();
+    var template = Handlebars.compile(source);
+    var newHTML = template(cities[i]);
+
+    $('.cities').append(newHTML);
+  }
+};
+renderCities();
+
+$('.search').on('click', function () {
+  var search = $('#search-query').val();
+  fetchForecast(search);
```

Comment:
You could move line 50 to line 7 and not have to repeat the click handler for this search button twice.